### ISSUE TITLE
Need testing-library

### ISSUE THREAD

**Issue by solenopsys** [vobyjs/voby#38]
I tried testing Solid with easy-dom, but events were not working.

After I tried testing with @testing-library/dom, it worked fine.

I needed a testing library for Voby.

Example for solid: https://github.com/solidjs/solid-testing-library/

**Comment by fabiospampinato** [comment]
I have no experience with this. If somebody wants to implement it we can bring it under the vobyjs github org 👍

**Comment by solenopsys** [comment]
I have started working here: https://github.com/solenopsys/converged-test. The code currently contains many unnecessary elements and requires substantial time for deep refactoring.

**Comment by fabiospampinato** [comment]
Is that supposed to be the equivalent for Voby of "solid-testing-library"? I'm not sure how these things should be written, I think it would probably be good with the rest of the ecosystem on this.

**Comment by solenopsys** [comment]
Yes, I created it for my project and later forked voby-testing-library after completing it. The most important aspect of https://github.com/solidjs/solid-testing-library/ is the tests. I have plans to refactor the tests and adapt them for voby. Regarding the API, I need to give it some thought.
